src/renderers/dom/client/__tests__/ReactDOMTreeTraversal-test.js

```diff
@@ -21,6 +21,11 @@ var ReactTestUtils = require('ReactTestUtils');
 var ARG = {arg: true};
 var ARG2 = {arg2: true};
 
+var PHASES = {
```

Comment:
Fixed. Pushed with `--force`, but it looks like new github's review tool doesn't understand it.